gnosis_scan_api.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import json
+import datetime
+import calendar
+from constants import REQUEST_TIMEOUT, SUCCESS_CODE
+
+
+def get_block_range(year, month, day):
+    date_time = datetime.datetime(year, month, day)
+    start_timestamp = int(calendar.timegm(date_time.timetuple()))
+    url = (
+        "https://api.gnosisscan.io/api?module=block&action=getblocknobytime"
+        + "&timestamp="
+        + str(start_timestamp)
+        + "&closest=after&apikey=YourApiKeyToken"
+    )
+    try:
+        response = requests.get(
+            url,
+            timeout=REQUEST_TIMEOUT,
+        )
+        if response.status_code == SUCCESS_CODE:
+            data = json.loads(response.text)
```

Comment:
```suggestion
            data = response.json()
```
and no include of the json library is required then.

Reply:
Nice!